files/LDC.md

```diff
@@ -2,35 +2,47 @@
 
-Besides simplified characters, the issue of character variants also needs to be addressed (note that this is different from orthographic character choice, which we will discuss in the the word entries section). To ensure the consistency while keeping complexities as low as possible, rime-cantonese takes advantage of an open-source library called Open Chinese Convert (OpenCC), which supports two-way conversions between different regional variants of Chinese characters. OpenCC maintains a standard for all Chinese character variants (we will call it “the OpenCC standard” or “OpenCC variants” from now on). The OpenCC standard character list keeps the most semantic contrasts between code points and servers as the baseline character variants. And with this baseline, the library can convert characters between Mainland Simplified, Hong Kong traditional, Taiwan traditional variants with this standard. We recommend accompanying the OpenCC library when using rime-cantonese in applications. For instance, when building a Cantonese transliteration module, one can generate the missing Simplified Chinese variants first with OpenCC in order to get a comprehensive character-to-pronunciation mapping list.
+We recommend accompanying the OpenCC library when using rime-cantonese in applications. For instance, when building a Cantonese transliteration module, one can generate the missing Simplified Chinese variants first with OpenCC in order to get a comprehensive character-to-pronunciation mapping list.
 
-## Multi-character word and phrase entries (word entries)
+### About variants Chinese characters
+
+Variant Chinese characters are Chinese characters that are homophones and synonyms. For example, 𥤚 is a variant character of 秋 (‘autumn’). These Chinese characters are not actively used in modern times. Thus, variant characters are included only in char entries and not in word entries. Particularly, some Japanese Shinjitai characters and Vietnamese Chữ Nôm are also included because they are more commonly found in daily life. For example, 円 is the Japanese variant for 圓 (“unit of currency”), while it is commonly used to describe the currency unit of Japan.
```

Comment:
autumn 雙引號
unit of currency 重複咗

Reply:
我merge咗先再改

Reply:
```suggestion
Variant Chinese characters are Chinese characters that are homophones and synonyms. For example, 𥤚 is a variant character of 秋 ("autumn"). These Chinese characters are not actively used in modern times. Thus, variant characters are included only in char entries and not in word entries. Particularly, some Japanese Shinjitai characters and Vietnamese Chữ Nôm are also included because they are more commonly found in daily life. For example, 円 is the Japanese variant for 圓, while it is commonly used to describe the currency unit of Japan.
```

Reply:
唔好整個咁深嘅字啦，iOS 顯示唔到，我仲以為漏咗
仲有雙引號用返 “ ”